packages/ra-ui-materialui/src/field/ChipField.tsx

```diff
@@ -17,7 +17,7 @@ const useStyles = makeStyles(
     { name: 'RaChipField' }
 );
 
-export const ChipField = memo<ChipFieldProps>((props: ChipFieldProps) => {
+export const ChipField: FC<ChipFieldProps> = props => {
```

Comment:
You forgot to maintain the `memo` call

Reply:
Originally was: 
`export const ChipField: FC<ChipFieldProps> = memo<ChipFieldProps>(props => {`